src/main/java/racingcargame/RacingCarGame.java

```diff
@@ -31,6 +32,28 @@ public void startGame(final RandomOption randomOption) {
         }
     }
 
+    public void showGameResult() {
+        sortDistance(cars);
+        for (int i = 0; i < cars.size(); i++) {
+            selectTie(cars, i);
+        }
+        printWinner(winners);
+    }
+
+    private void sortDistance(List<RacingCar> cars) {
+        Collections.sort(cars, Collections.reverseOrder());
```

Comment:
아래와 같이 표현도 가능하네요 :-) 
```suggestion
      cars.sort(Collections.reverseOrder());
```